BackofficeBundle/EventSubscriber/DataTransformer/ArrayToStringTransformer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer;
+
+/**
+ * Class ArrayToStringTransformer
+ */
+class ArrayToStringTransformer implements ValueTransformerInterface
+{
+    /**
+     * @param array $data
+     *
+     * @return string
+     */
+    public function transform($data)
+    {
+        $output = $data;
+        if (is_array($data)) {
+            $output = "<ul>";
+            foreach ($data as $item) {
+                $output .= "<li>" . $this->transform($item) . "</li>";
+            }
+            $output .= "</ul>";
+        }
```

Comment:
Plus the datatransformer dir is missnamed : it's not a true datatransformer as there's no reversetransform, and they are not implicated in form process. Instead they are strategies used to display complex things as html string.

So I think instead of datatransfomer, we should have a coktail of 'strategy' + 'htmlString' + 'tranformer'